cmake/developer_package/download/download_and_extract.cmake

```diff
@@ -148,14 +153,16 @@ function (DownloadOrExtractInternal URL archive_path unpacked_path folder fattal
 
 endfunction(DownloadOrExtractInternal)
 
-function (CheckOrDownloadAndExtract component RELATIVE_URL archive_name unpacked_path result_path folder fattal resultExt use_alternatives sha256 files_to_extract)
+function (CheckOrDownloadAndExtract component RELATIVE_URL archive_name unpacked_path result_path folder fattal resultExt use_alternatives sha256 files_to_extract USE_NEW_LOCATION)
   set (archive_path ${TEMP}/download/${archive_name})
   set (status "ON")
 
   if(DEFINED IE_PATH_TO_DEPS)
     set(URL "${IE_PATH_TO_DEPS}/${RELATIVE_URL}")
   elseif(DEFINED ENV{IE_PATH_TO_DEPS})
     set(URL "$ENV{IE_PATH_TO_DEPS}/${RELATIVE_URL}")
+  elseif(USE_NEW_LOCATION)
+    set(URL "https://storage.openvinotoolkit.org/dependencies/${RELATIVE_URL}")
   else()
     set(URL "https://download.01.org/opencv/master/openvinotoolkit/${RELATIVE_URL}")
```

Comment:
why do we need new location?

Reply:
As @artanokhov explained, there is no write access to 01.org anymore, so we need to change the library location to Amazon S3 ([https://storage.openvinotoolkit.org)](https://storage.openvinotoolkit.org/).
See the ticket (92208) for more details.

Reply:
We can see that this PR https://github.com/openvinotoolkit/openvino/pull/12690/files changes to archives on 01org. So, it's accessible.